de.dlr.sc.virsat.server.feature/feature.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="de.dlr.sc.virsat.server.feature"
+      label="VirSat Core Headless Rest Jetty Server feature"
+      version="4.10.0.qualifier"
+      provider-name="DLR (German Aerospace Center)"
+      plugin="de.dlr.sc.virsat.branding.ui"
+      license-feature="de.dlr.sc.virsat.license.feature">
+
+   <description>
+      VirSat Team Connector
```

Comment:
Probably should be VirSat Server or something. Looks like copy paste.

Reply:
I think this is fixed already in a different branch. I will check it.